extensions/cognigy/eslint.config.js

```diff
@@ -0,0 +1,56 @@
+const js = require("@eslint/js");
```

Comment:
That's a very big and opinionated eslint config :D I would also assume that the rules specified here (or initially in the tslint config) might be outdated and might not reflect the current standards. Let's try the most simple default recommended ruling: https://typescript-eslint.io/getting-started/#step-2-configuration . Please note that the config file is named `eslint.config.mjs` in the example, not sure if it has relevance other than module.exports vs export default

```js filename="eslint.config.mjs"
import eslint from '@eslint/js';
import tseslint from 'typescript-eslint';

export default tseslint.config(
  eslint.configs.recommended,
  tseslint.configs.recommended,
);
```